pangram/pangram.py

```diff
@@ -0,0 +1,10 @@
+from string import ascii_lowercase as alphabet
+
+
+def pangram(sentence):
+
+    sentence_letters_in_alphabet = set(
+        letter.lower() for letter in sentence if letter.lower() in alphabet
```

Comment:
I believe this:

```
letter.lower() for letter in sentence if letter.lower() in alphabet
```

is equivalent to this:

```
letter for letter in sentence.lower() if letter in alphabet
```

You can just call `.lower()` on the original sentence.